src/Spreadaccacadmey/SpreadAcademy.styled.js

```diff
@@ -0,0 +1,205 @@
+import styled from "styled-components";
+
+export const SpreadAcademyContainer = styled.section`
+  display: flex;
+  justify-content: center;
+  background-color: #fff6f6;
+  align-items: center;
+  @media (max-width: 700px) {
+    justify-content: flex-start;
+  }
+`;
+export const SpreadAcademySection = styled.section`
+  background-color: #fff6f6;
+`;
+
+export const SpreadAcademyWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 71px 0px 0px;
+  @media (max-width: 700px) {
+    flex-direction: column;
+    padding: 0px 0px 0px 0px;
+  }
+`;
+
+export const Pstyle = styled.p`
+  font-size: 13px;
+`;
+
+export const LeftWrapper = styled.div`
+  max-width: max-content;
+  height: auto;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  @media (max-width: 700px) {
+    padding: 7px 19px 64px 14px;
+  }
+`;
+
+export const IconContainar = styled.div`
+  background-color: #fff6f6;
+  margin: 23px 0px;
+  height: auto;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+`;
+
+export const RightCardImage = styled.img`
+  max-width: 278px;
+  z-index: 1;
+  border-radius: 16px;
+  @media (max-width: 700px) {
+    max-width: 178px;
+  }
+`;
+
+export const SmallCard = styled.div`
+  position: absolute;
+  top: -17px;
+  right: -3px;
+  background: rgb(254, 79, 82);
+  border-radius: 3px;
+  padding: 7px 1px 5px 7px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  max-width: 260px;
+  width: 100%;
+  z-index: 2;
+  color: white;
+  @media (max-width: 700px) {
+    padding: 11px 12px 11px 11px;
+    right: -17px;
+    width: 74%;
```

Comment:
not this use max-width as on uppr side you used max width @PRABgit01

Reply:
ok

Reply:
done